### ISSUE TITLE
Method for changing particle colors with resource pack

### ISSUE THREAD

**Issue by eightsevenfive** [sp614x/optifine#3487]
Hello, I'm making a resource/texture pack, and I'm trying to change the color of the critical hit particle. The texture file (critical_hit.png) is grayscale and colored to a tan or beige by the game. The sample configuration for color.properties says nothing about recoloring this particle, just water and portal particles. I experimented by entering particle.critical_hit=FF0000 into color.properties, but it had no effect. Since Optifine supports changing so many other hardcoded colors (swamp, potions, etc.) I figured there might by a way to change critical sparks that I'm overlooking.

Is this just a particle which hasn't custom color support added to Optifine, or am I missing something? Thanks for any help. :)

**Comment by UltravioletBag** [comment]
yeah id really like an answer/ addition to solve this problem. I have the exact same issue but with enchanted_hit particles. id really like to change the color from the hardcoded green.  please help lol.

**Comment by thebruhmoment** [comment]
bump
